File: easy/Multiply-Lists/Multiply-Lists.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here
            ShowNums(line);
        }
    }
    
    public static void ShowNums(String line){
        int pos = line.indexOf("|");
        String[] numsStr1 = line.substring(0,pos).trim().split(" ");
        String[] numsStr2 = line.substring(pos+1).trim().split(" ");
        int leng = numsStr1.length;
        for(int i=0;i<leng;i++){
            System.out.print(Integer.valueOf(numsStr1[i])
                            *Integer.valueOf(numsStr2[i]) + " ");
        }
        System.out.println();
        
        
    }
}

File: easy/Multiply-Lists/Multiply Lists.cs
using System;
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
            // do something with line
            ShowNums(line);
        }
    }
    
    public static void ShowNums(string line){
        int pos = line.IndexOf('|');
        string[] numsStr1 = line.Substring(0,pos).Trim().Split(' ');
        string[] numsStr2 = line.Substring(pos+1).Trim().Split(' ');
        int leng = numsStr1.Length;
        for(int i=0;i<leng;i++){
            Console.Write(Convert.ToInt32(numsStr1[i])
                            *Convert.ToInt32(numsStr2[i]) + " ");
        }
        Console.WriteLine();
        
        
    }
}
